### ISSUE TITLE
MTRs on capital income

### ISSUE THREAD

**Issue by jdebacker** [TPRU-India/OG-India#19]
When I compute the weighted average marginal tax rate on capital income in `get_micro_data.py` using the `pitSmallData.csv`, I'm getting a weighted average marginal tax rate on capital income that exceeds 100% for most tax payers.

Looking into the underlying data, no filer has an MTR for any capital  income source above 36%:

```
In [25]: df[['STCG 1 MTR', 'STCG 2 MTR', 'STCG AMT']].describe()
Out[25]:
         STCG 1 MTR    STCG 2 MTR      STCG AMT
count  10000.000000  10000.000000  10000.000000
mean       0.143129      0.286303      0.191846
std        0.044887      0.089743      0.120481
min        0.000000      0.000000      0.000000
25%        0.156000      0.312000      0.052000
50%        0.156000      0.312000      0.208000
75%        0.156000      0.312000      0.312000
max        0.179400      0.358800      0.358800

In [26]: df.keys()
Out[26]:
Index(['STCG 1 MTR', 'STCG 2 MTR', 'STCG AMT', 'LT CG 1', 'LT CG 2',
       'Not Horse'],
      dtype='object')

In [27]: df[['LT CG 1', 'LT CG 2', 'Not Horse']].describe()
Out[27]:
            LT CG 1       LT CG 2  Not Horse
count  10000.000000  10000.000000    10000.0
mean       0.095404      0.190853        0.0
std        0.029944      0.059836        0.0
min        0.000000      0.000000        0.0
25%        0.104000      0.208000        0.0
50%        0.104000      0.208000        0.0
75%        0.104000      0.208000        0.0
max        0.119600      0.239200        0.0
```

There must be an issue computing this weighted average.  It should be fixed as it's preventing any subsequent analysis after tax functions are estimated.

**Comment by rickecon** [comment]
This is an Indian [`taxcalc`](https://github.com/TPRU-India/taxcalc) issue that needs to be resolved by an update to the `taxcalc` repository. @sebastiansajie.

**Comment by jdebacker** [comment]
@rickecon This is not a `taxcalc` issue.  It has to do with something in the weighted average computed in `ogindia.get_micro_data.py`.  I'm looking into it.  But from the results posted above, the rates direction from `taxcalc` look reasonable.

**Comment by jdebacker** [comment]
This issue was addressed with PR #20
